explorer/interpreter/value.cpp

```diff
@@ -698,6 +698,23 @@ void Value::Print(llvm::raw_ostream& out) const {
   }
 }
 
+void IntrinsicConstraint::Print(llvm::raw_ostream& out) const {
+  out << *type << " is ";
+  switch (kind) {
+    case IntrinsicConstraint::ImplicitAs:
+      out << "__intrinsic_implicit_as";
+      break;
+  }
+  if (!arguments.empty()) {
+    out << "(";
+    llvm::ListSeparator comma(", ");
```

Comment:
```suggestion
    llvm::ListSeparator comma;
```

I'm happy to use the default constructor for this, I think most of the code already does.